### ISSUE TITLE
Breakpoints creates a duplicate slide after screen is resize

### ISSUE THREAD

**Issue by coopersamuel** [antoniandre/vueper-slides#40]
Hey @antoniandre,

Great component. I'd love to use it in my app if this bug gets fixed. Here's some code with the problem:
```
<vueper-slides
    :visible-slides="3"
    slide-multiple
    :slide-ratio="1/4"
    :dragging-distance="200"
    :breakpoints="{ 1500: { visibleSlides: 2 }, 1050: { visibleSlides: 1 } }">
    <vueper-slide v-for="i in 5" :key="i" :title="i.toString()"></vueper-slide>
  </vueper-slides>
```

As you can see, I'm changing the amount of visible slides depending on the screen size. It works well, except I'm seeing an extra slide (the last one) being duplicated and put in at the beginning.

I've modified the example codepen to show this problem, the arrows aren't showing up for some reason, but whatever - https://codepen.io/anon/pen/LwPMJw

Exact steps to repro with the codepen:
- Reduce your browser window until only one slide is showing
- (Notice that for some reason it scrolls to slide 2)
- Scroll to the right through the entire slide deck (until you see slide 1)
- (Notice the blank slide in there)
- Increase your browser window and notice that slide 5 has been duplicated and placed at the beginning before slide 1

Please let me know if you can get to this soon, I'd really like to use this component 😄

**Comment by antoniandre** [comment]
Hi @coopersamuel, Thank you for your feedback and reporting this bug.
ATM I have to finish an urgent task on another project but I will have a look at this end of next week.

**Comment by coopersamuel** [comment]
Hey @antoniandre, sounds great, I'll monitor this thread for updates

**Comment by coopersamuel** [comment]
@antoniandre any traction on this?

**Comment by antoniandre** [comment]
Hi @coopersamuel,
I've been quite busy for some time, but today I have released Vueper Slides 2.0 with a bug fix for that among other great improvements!
Please check the release notes in the documentation if you are still keen to use this lib,
Here is a fixed demo of your issue: https://codepen.io/antoniandre/pen/OJJGbLL?editors=1010

Cheers.

**Comment by antoniandre** [comment]
This issue is fixed. Closing due to inactivity.
